<?xml version="1.0" encoding="utf-8"?>
<launch>
  
  <!-- Start the Sawyer pick and place demo -->
  <node pkg="planning_sim_demo" type="ik_pick_and_place_demo.py" name="ik_pick_and_place_demo" output="screen"/>

</launch>
